src/GdbConnection.h

```diff
@@ -197,7 +204,14 @@ struct GdbRequest {
     std::vector<GdbContAction> actions;
   } cont_;
   std::string text_;
-
+  struct Tls {
+    size_t offset;
+    remote_ptr<void> load_module;
+  } tls_;
+  struct Symbol {
+    std::string address;
```

Comment:
I did it this way so that GdbServer can easily see when there is no address.  Using a sentinel address is possible but seemed less clean.  I suppose I could stick a bool in there as well.